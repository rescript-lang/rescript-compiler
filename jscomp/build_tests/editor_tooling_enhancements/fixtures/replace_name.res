type x = {
  name: string,
  age: int,
}

let x: x = {
  name: "hello",
  agee: 10,
}
