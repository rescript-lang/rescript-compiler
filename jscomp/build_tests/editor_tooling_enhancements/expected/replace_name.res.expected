
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/replace_name.res[0m:[2m8:3-6[0m

   6 [2mâ”‚[0m let x: x = {
   7 [2mâ”‚[0m   name: "hello",
   [1;31m8[0m [2mâ”‚[0m   [1;31magee[0m: 10,
   9 [2mâ”‚[0m }
  10 [2mâ”‚[0m 

  The field [1;31magee[0m does not belong to type [1;33mx[0m
  
  This record expression is expected to have type [1;33mx[0m
Hint: Did you mean age?
=== CODE ACTIONS ===
[{"title": "Replace with `agee`", "kind": "quickfix" "loc": {"start": {"line": 8, "col": 74},"end": {"line": 8, "col": 78}}, "type": "replaceWith", "replaceWith": "agee"}]